services/freeze_type.proto

```diff
@@ -19,53 +27,107 @@ package proto;
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- * ‍
  */
 
 option java_package = "com.hederahashgraph.api.proto.java";
 // <<<pbj.java_package = "com.hedera.hapi.node.freeze">>> This comment is special code for setting PBJ Compiler java package
 option java_multiple_files = true;
 
 /**
-* The type of network freeze or upgrade operation to be performed. This type dictates which 
-* fields are required. 
-*/
+ * An enumeration of possible network freeze types.
+ *
+ * Each enumerated value SHALL be associated to a single network freeze
+ * scenario. Each freeze scenario defines the specific parameters
+ * REQUIRED for that freeze.
+ */
 enum FreezeType {
     /**
-     * An (invalid) default value for this enum, to ensure the client explicitly sets 
-     * the intended type of freeze transaction.
+     * An invalid freeze type.
+     * <p>
+     * The first value in a protobuf enum is a default value. This default
+     * is RECOMMENDED to be an invalid value to aid in detecting unset fields.
      */
     UNKNOWN_FREEZE_TYPE = 0;
 
     /**
-     * Freezes the network at the specified time. The start_time field must be provided and 
-     * must reference a future time. Any values specified for the update_file and file_hash 
-     * fields will be ignored. This transaction does not perform any network changes or 
-     * upgrades and requires manual intervention to restart the network. 
+     * Freeze the network, and take no further action.
+     * <p>
+     * The `start_time` field is REQUIRED and SHOULD be between
```

Comment:
`start_time` should be any time after consensus time.

Reply:
The caller has little ability to predict consensus start, so a recommendation seemed appropriate, and this was provided when the question "how long after transaction valid start might consensus time be" and "how long before a freeze is the transaction typically sent".

I've adjusted to clarify both recommended and required values.